src/app/reports/report-card.component.ts

```diff
@@ -0,0 +1,166 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { DomSanitizer } from "@angular/platform-browser";
+
+import { MessagingService } from "jslib-common/abstractions/messaging.service";
+import { StateService } from "jslib-common/abstractions/state.service";
+
+export enum ReportTypes {
```

Comment:
I'm confused by this enum - can you help clarify its purpose?

Reply:
It's used in the `ReportListComponent`. And as a compiler check for the report type.